pkg/plugin/conntrack/_cprog/conntrack.c

```diff
@@ -174,6 +179,8 @@ static __always_inline bool _ct_handle_tcp_connection(struct packet *p, struct c
     if (CT_CONNECTION_LIFETIME_TCP > UINT32_MAX - now) {
         return false;
     }
+    // Set the connection as unknown direction since we did not capture the SYN packet.
+    new_value.is_direction_unknown = true;
```

Comment:
It doesn't look like this ever becomes explicitly false. Am I missing something?

Reply:
instead of adding a new field, let reuse the existing `traffic_direction`?

Reply:
As discussed with @nddq, we agree to use a separate flag to avoid modifying conntrack behavior. This new flag will allow us to create a metric to then monitor the number of connections with unknown direction and the changes over time.